### DISCUSSION TITLE
On clearing callsign textfield in live QSO clear Name, Location, Gridsquare and comment as well

### DISCUSSION THREAD

**Discussion by PA4RB** [magicbug/Cloudlog#1819] [Ideas / Feature Requests]
Often i don't manage to get a contact in the log (Heey QRP) . I then fill a new callsign in in the textfield box on the live QSO page. On clearing Callsign textfield in live QSO the Name, Location, Gridsquare and comment fields should be cleared as well. 

In the screenshot you'll see the info for G6INU but i have overwritten the callsign field with GM5WS.

![clear textfields](https://user-images.githubusercontent.com/60810525/205061365-3d752dbf-5d36-45d1-846f-f249d838a14e.png)

**Comment by magicbug** [comment]
This is happening as the lookup as already been called and will display regardless, need to add a check that the Callsign is still the one that was looked up, fix coming soon
